gatsby-config.js

```diff
@@ -19,6 +19,27 @@ module.exports = {
                 path: `${__dirname}/src/docs`
             }
         },
+        {
+            resolve: `gatsby-plugin-manifest`,
+            options: {
+                name: `Gitpod`,
+                short_name: `Gitpod`,
+                start_url: `/`,
+                background_color: `#f7f0eb`,
+                theme_color: `#a2466c`,
```

Comment:
Maybe set the theme colour to the blue that we use as our primary colour on the website as this does not look that great IMO.

![image](https://user-images.githubusercontent.com/46004116/82164725-706a0200-98cb-11ea-8618-6416b71e402b.png)
```suggestion
                theme_color: `#1aa6e4`,
             
```